sdk/compute/armcompute/zz_generated_connection.go

```diff
@@ -0,0 +1,82 @@
+// +build go1.13
```

Comment:
Why this file is generated? We have `azure-arm: true`

Reply:
It really seems like this setting is not being included.  Are you sure the new way of building is including the correct config files?